usaspending_api/reporting/v2/views/agencies/toptier_code/overview.py

```diff
@@ -193,7 +193,7 @@ def format_result(self, result):
                     "percent_of_total_budgetary_resources": round(
                         result["total_budgetary_resources"] * 100 / result["gtas_total_budgetary_resources"], 2
                     )
-                    if result["gtas_total_budgetary_resources"]
+                    if result["gtas_total_budgetary_resources"] and result["total_budgetary_resources"]
```

Comment:
Both `unlinked_contract_award_count` and `unlinked_assistance_award_count` are affected by similar issues.